packages/diagrams/lib/src/scaffold.dart

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:diagram_capture/diagram_capture.dart';
+import 'package:flutter/material.dart';
+
+import 'diagram_step.dart';
+
+class ScaffoldDiagram extends StatefulWidget implements DiagramMetadata {
+  @override
+  final String name = 'scaffold';
+
+  @override
+  State<StatefulWidget> createState() => ScaffoldDiagramState();
+}
+
+class ScaffoldDiagramState extends State<ScaffoldDiagram> {
+  int _count = 0;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
```

Comment:
I might put this in a constrained box or something similar to enforce a typical size and layout.

This diagram could also be incredibly informative by using some of the other features of the Scaffold class and labelling them. You could even create more than one diagram to illustrate the different features as well!

Reply:
Good call, fixed!